### ISSUE TITLE
🐛 Bug Report: Can't Create Document w/ Attribute w/ Period

### ISSUE THREAD

**Issue by stnguyen90** [appwrite/appwrite#4326]
### 👟 Reproduction steps

1. Create a collection with an attribute containing a period (.)
2. Create a document in that collection

### 👍 Expected behavior

The document is created

### 👎 Actual Behavior

A 400 error occurs:

My attribute was `djd.djdj`

```json
{
    "message": "Invalid document structure: Unknown attribute: \"djddjdj\"",
    "code": 400,
    "type": "document_invalid_structure",
    "version": "1.0.1"
}
```

### 🎲 Appwrite version

Version 1.0.x

### 💻 Operating system

Linux

### 🧱 Your Environment

Tested on Appwrite 1.0.1.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
Previous issue: https://github.com/appwrite/appwrite/issues/3033

**Comment by Akshay-Rana-Gujjar** [comment]
@tessamero please assign this issue to me. Thanks

**Comment by tessamero** [comment]
@Akshay-Rana-Gujjar , thanks for your interest! You may work on this issue 🙏 Happy hacking! 🎃

**Comment by abnegate** [comment]
@Akshay-Rana-Gujjar Did you manage to make some progress on this task?

**Comment by abnegate** [comment]
Fixed in 1.3.0

**Comment by Akshay-Rana-Gujjar** [comment]
> Fixed in 1.3.0

Sorry, I missed you comment (asking for progress) and today got your comment notification.
